pkg/kubectl/describe.go

```diff
@@ -1395,6 +1396,32 @@ func DescribeEvents(el *api.EventList, w io.Writer) {
 	}
 }
 
+// DeploymentDescriber generates information about a deployment.
+type DeploymentDescriber struct {
+	client.Interface
+}
+
+func (dd *DeploymentDescriber) Describe(namespace, name string) (string, error) {
+	d, err := dd.Experimental().Deployments(namespace).Get(name)
```

Comment:
We should get the replication controllers managed by this dep and show them in some order.  If they can be loaded we shouldn't hard fail (show error and continue)